source/extensions/filters/http/ext_proc/processor_state.cc

```diff
@@ -66,6 +69,9 @@ bool ProcessorState::handleBodyResponse(const BodyResponse& response) {
     modifyBufferedData([this, &response](Buffer::Instance& data) {
       MutationUtils::applyCommonBodyResponse(response, headers_, data);
     });
+    if (response.response().clear_route_cache()) {
+      filter_callbacks_->clearRouteCache();
```

Comment:
Since routing doesn't use the body, does it make sense to clear here?

Reply:
When the body is in buffered mode, it's possible to set headers when responding to the body callback, so it's possible that someone will want this.